src/multivariate.jl

```diff
@@ -1,3 +1,91 @@
+## Dirichlet ##
```

Comment:
I'm a bit surprised, why is this part of this PR?

Reply:
Yes sorry this leaked from another branch. I will remove it.